### DISCUSSION TITLE
Best way to upgrade ?

### DISCUSSION THREAD

**Discussion by JohanAlvedal** [JensenNick/huawei_solar_pees#39] [Question / Answer] [answered]
what is the best way to upgrade from a previous version without starting from scratch?

**Comment by JensenNick** [comment, marked as answer]
There is no change to existing sensor names, so it should be as simple as just to replace your existing `huawei_solar_pees.yaml` file with the new version (v.2) and add the `huawei_solar_input.yaml` file to the package folder. Create the input card and restart. Once restarted set your specific inputs in the input card. If you do not wish to use the efficiency corrected power input sensor just leave the boolean switches off (and there is no need to specify your inverter).
